server/src/alda/now.clj

```diff
@@ -93,8 +93,9 @@
    evaluating alda.lisp code, e.g. (score (part 'piano' (note (pitch :c)))),
    or an atom referencing such a map."
   [score & [play-opts]]
-  (sound/with-play-opts (merge {:async? true} play-opts)
-    (sound/play! (if (instance? clojure.lang.Atom score)
-                   @score
-                   score))))
+  (println "testing alda dev setup"))
```

Comment:
I can explain what's going on here -- since 1.0.0-rc35, the server doesn't execute that part of the code (the "playing audio") part anymore. Now we have separate worker processes that handle parsing, evaluating and playing Alda scores.

Now that there are more moving parts, the setup is slightly more complicated, but hopefully not too much so... I added [some info](https://github.com/alda-lang/alda/blob/master/doc/development-guide.md#alda-worker) to the development guide about starting server and worker processes for development.

There are step by step instructions in the link above, but basically, the new way to do it is to:

- Start a development server with the `ALDA_DISABLE_SUPERVISOR` environment variable set (this keeps the server from inadvertently killing your debug worker process) and the `--workers 0` option so it doesn't start any worker processes of its own.
- Start a development worker process with `ALDA_DEBUG` environment variable set and the `--port 12345` option, where `12345` is the _backend port_ where the server communicates with its workers. (The server logs that port number out when it starts.)
- If you have this working correctly, you'll start to see the worker logging "Got HEARTBEAT from server" about once per second. Then when you send a request to the server by running e.g. `alda play -c 'piano: c'`, the server will forward the request to your worker (because it's the only one available) and you should see `testing alda dev setup` printed in the output of the worker.